lib/ansible_tower_client/base_models/project.rb

```diff
@@ -17,5 +17,13 @@ def update
 
       api.project_updates.find(update['project_update'])
     end
+
+    def last_update_stdout(format = 'txt')
+      return if !related.respond_to?(:last_update) or related['last_update'].nil?
```

Comment:
I believe `or` and `||` are the same in this case... Generally, `||` is preferred except in the very specific cases where `or` might be helpful.  I still can't understand `or` most times and stay away from it.  http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/